keyboards/bastardkb/charybdis/charybdis.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Quentin LEBASTARD <qlebastard@gmail.com>
+ * Copyright 2021 Charly Delay <charly@codesink.dev> (@0xcharly)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#if defined(KEYBOARD_bastardkb_charybdis_3x5)
+#include "3x5/3x5.h"
+#elif defined(KEYBOARD_bastardkb_charybdis_4x6)
+#include "4x6/4x6.h"
```

Comment:
Should be automatically pathed, so only needs the file name.
```suggestion
#include "3x5.h"
#elif defined(KEYBOARD_bastardkb_charybdis_4x6)
#include "4x6.h"
```